src/main/java/frc/robot/misc/Constants.java

```diff
@@ -38,16 +41,16 @@ public static final class Swerve {
         /* Swerve Current Limiting */
         public static final int angleContinuousCurrentLimit = 20;
         public static final int driveContinuousCurrentLimit = 40;
-
+        
         /* Angle Motor PID Values */
-        public static final double angleKP = 1.6; //1.6 on our robot
-        public static final double angleKI = 1.8; //1.8 on our robot
-        public static final double angleKD = 0.0;
-        public static final double angleKFF = 1.1;
+        public static final double angleKP = 2;//12;//1.6; //1.6 on our robot
```

Comment:
Just clean up all the messy comments at the end. If anything, put a comment above the line that explains the other values and how you got to 2.